src/tbb/scheduler_common.h

```diff
@@ -513,9 +566,8 @@ class alignas (max_nfs_size) task_dispatcher {
         }
 
         for (auto& elem : m_reference_vertex_map) {
-            d1::reference_vertex*& node = elem.second;
-            node->~reference_vertex();
-            cache_aligned_deallocate(node);
+            thread_reference_vertex*& node = elem.second;
+            node->release_ownership();
```

Comment:
It seems like we don't need a `node` variable any more. 
```suggestion
            elem.second->release_ownership();
```
In contrast, having a type here may add useful information, so feel free to keep as-is.

Reply:
Decided to keep as-is to be more consistent with the cleanup in `get_thread_reference_vertex`.